src/components/Card/index.tsx

```diff
@@ -36,6 +36,14 @@ const useStyles = createUseStyles({
     }
 });
 
+const useLabelStyles = createUseStyles({
+    required: {
```

Comment:
This class should be taken from the theme, like for example the Login component does this here: https://github.com/softwaregroup-bg/ut-prime/blob/d523858b0721a6f4aee20ec3239aabfc365b718e/src/components/Login/index.tsx#L249

The theme classes are defined in impl-xxx/browser/theme/classes.css